docs/lua-modules/httpserver.md

```diff
@@ -42,14 +46,15 @@ Callback function has 2 arguments: `req` (request) and `res` (response). The fir
 	- `name`: Header name
 	- `value`: Header value
 
-- `ondata`: value to setup handler function HTTP data. Handler function has 2 parameters:
+- `ondata`: value to setup handler function HTTP data. Handler function has 3 parameters:
 	- `self`: `req` object
 	- `chunk`: Request data
+	- `last`: set true while the last chunk
```

Comment:
As mentioned earlier, I think I'd prefer that `chunk` be `nil` for the end of the stream.  The documentation could still stand an update, even so.